apps/metronome/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { InstantApp } from "@felvin-search/core";
+import { Component, queryToData } from "./App";
+
+const App: InstantApp = {
+  id: "@felvin-search-apps/metronome",
+  name: "metronome",
+  description: "This app plays a metronome with a set bpm",
+  queryToData,
+  Component,
+  screenshotPath: "./files/screenshot.png",
```

Comment:
It's good that you uncommented these lines, can you also go ahead and add a screenshot of the app.